integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -123,6 +123,37 @@ pub(crate) fn produce_blocks_from_height(
     produce_blocks_from_height_with_protocol_version(env, blocks_number, height, PROTOCOL_VERSION)
 }
 
+#[allow(unused)]
+pub(crate) fn create_account(
```

Comment:
Why do you need this function? I think it's better to only add it when needed to reduce code bloat.

Reply:
Switched to conditional compilation. Hopefully that is more clear